include/sys/util.h

```diff
@@ -119,6 +120,56 @@ static inline s64_t arithmetic_shift_right(s64_t value, u8_t shift)
 	return (value >> shift) | (sign_ext << (64 - shift));
 }
 
+/**
+ * @brief      Convert a single character into a hexadecimal nibble.
+ *
+ * @param[in]  c     The character to convert
+ * @param      x     The address of storage for the converted number.
+ *
+ *  @return Zero on success or (negative) error code otherwise.
+ */
+int char2hex(char c, u8_t *x);
+
+/**
+ * @brief      Convert a single hexadecimal nibble into a character.
+ *
+ * @param[in]  c     The number to convert
+ * @param      x     The address of storage for the converted character.
+ *
+ *  @return Zero on success or (negative) error code otherwise.
+ */
+int hex2char(u8_t x, char *c);
+
+/**
+ * @brief      Convert a binary array into string representation.
+ *
+ * @param      hex     Address of where to store the string representation.
```

Comment:
Not sure if sanitychecker complains about these but the in the function documentation, the params are now in wrong order.

Reply:
Right, thank you. Updated.